icubam/db/test_gsheets.py

```diff
@@ -1,18 +1,26 @@
 from icubam import config
 from icubam.db import gsheets
 from absl.testing import absltest
+import os
+import unittest
 
 
-class SQLiteDBTest(absltest.TestCase):
+class GsheetsTest(absltest.TestCase):
+  gsheets_tokens_def = all([os.environ.get(key, False)
+                           for key in ['SHEET_ID', 'TOKEN_LOC']])
```

Comment:
I would have though that this should be defined one line before the class, otherwise it would have to be accessed as `self.gsheets_tokens_def` which is not possible from a decorator.

Although this seems to pass tests, and the two tests are indeed skipped, so maybe it's fine.